test/kubernetes_unittest.cc

```diff
@@ -13,6 +14,26 @@ class KubernetesTest : public ::testing::Test {
       throw(json::Exception) {
     return reader.GetNodeMetadata(node, collected_at, is_deleted);
   }
+
+  MetadataUpdater::ResourceMetadata GetClusterMetadata(
+      const KubernetesReader& reader, Timestamp collected_at) const
+      throw(json::Exception) {
+    return reader.GetClusterMetadata(collected_at);
+  }
+
+  void UpdateServiceToMetadataCache(
+      KubernetesReader& reader, const json::Object* service,
```

Comment:
Let's prefer pointers to non-const references ([style guide](https://google.github.io/styleguide/cppguide.html#Reference_Arguments)). Also in `UpdateServiceToPodsCache`.

Reply:
done